### PR TITLE
[#62] Added support for building docs for different fork of repo (master)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

**Comment by trel** [comment]
should we just pass in a git_repository... ?

and then it could be a full URL, or a local path... ?

**Comment by trel** [comment]
MAKEIRODSREPOSITORY

**Comment by korydraughn** [comment]
could it really be a local path?  supporting local repos would require more changes to make that work, right?

**Comment by korydraughn** [comment]
do we want to add some underscores in the variable names? they are a bit hard to read.

Sticking with https://github/${GITHUB_ACCT}/irods somewhat guarantees that the repo being cloned is a fork of the irods/irods repo, right?

File: README.md [hunk 1]
```diff
@@ -25,7 +26,7 @@ Build
 The `Makefile` default target will build two other targets, 'doxygen' and 'mkdocs'.
 
 ```
-$ make
+$ make [MAKEGITHUBACCOUNT=<value>] [MAKEIRODSVERSION=<value>]
```
Comment (trel) [inline README.md:29]: change this to the default value here?

**Pull request merged**